### ISSUE TITLE
[Accessibility]: File decoration <meter> element aria-label not announced by VoiceOver

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36857]
### Audit type

Screen reader navigation

### User journey audit issue

#34060

### Problem description

Extensions can contribute file decorations that render `<meter>` elements after a file in the file tree. VoiceOver doesn't announce a file decoration's state.

### Expected behavior

All screen readers should announce `<meter>` element labels and state. However, [many screen reader + browser combinations do not support the `<meter>` element](https://scottaohara.github.io/a11y_styled_form_controls/src/meter/#:~:text=VoiceOver%20will%20completely%20ignore%20the,they%20are%20styled%20or%20not.&text=Chrome%20%2B%20TalkBack%20will%20completely%20ignore,%2Dlabel%20or%20aria%2Dlabelledby%20.). 

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
